test/coverage.js

```diff
@@ -175,6 +175,14 @@ describe('Coverage', () => {
         done();
     });
 
+    it('retains original value of conditional result with comma operator', (done) => {
```

Comment:
Not sure what this means. I copied the test above, and just modified it. What do I need to do differently? 
Note, the purpose of this PR is not to add new or missing functionality. If coverage of conditional operator and comma operator was wrong before, it will continue to be wrong after this. This PR just to fix a bug which causes errors (in my case tests starts to fail) when lab is run with coverage switched on.